clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/model/EurekaInstance.groovy

```diff
@@ -97,7 +99,8 @@ class DiscoveryInstance implements Health {
       vipAddress,
       secureVipAddress,
       lastUpdatedTimestamp,
-      asgName)
+      asgName,
+      metadata?.titusTaskId)
```

Comment:
still feels like it would be nice to not have titusTaskId defined in here (and I'd be ok with removing all the AWS specific bits in 'dataCenterInfo.metadata' too) - maybe we just add a metadata attribute to EurekaInstance that is set to dataCenterInfo.metadata + metadata ?  Since it gets converted back to a map when passed to the provider interface it should mostly work the same but with less cloud-provider specific opinions in this module.

wdyt?